media_ui/src/main/java/edu/artic/media/ui/AudioDetailsFragment.kt

```diff
@@ -165,15 +170,12 @@ class AudioDetailsFragment : BaseViewModelFragment<AudioDetailsViewModel>() {
         viewModel.relatedTours
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe { tours ->
-                    if (tours.isEmpty()) {
-                        relatedTourTitle.visibility = View.GONE
-                        relatedToursView.visibility = View.GONE
-                        dividerBelowRelatedTours.visibility = View.GONE
-                    } else {
-                        relatedToursView.removeAllViews()
-                        relatedTourTitle.visibility = View.VISIBLE
-                        relatedToursView.visibility = View.VISIBLE
-                        dividerBelowRelatedTours.visibility = View.VISIBLE
+                    val hasData = tours.isNotEmpty()
+                    relatedTourTitle.show(show = hasData)
+                    relatedToursView.show(show = hasData)
+                    dividerBelowRelatedTours.show(show = hasData)
+
+                    if (hasData) {
```

Comment:
Just to clarify, we do not need to call `relatedToursView.removeAllViews()` anymore?

Reply:
My bad. That def should be there. Thank you for catching it @Cliabhach